andela/mrm_api
# #164266567 (v2) Add Structure id to Location model

#### What does this PR do?
- Add structure_id to Location Model
### Description of Task to be completed?

- [x] add `structure_id` foreign key to `Location` Model
- [x] Add OfficeStructure/Location relationships
- [x] Update create location mutation
- [x] Update location fixtures
- [x] Update base.py tests setup

#### How should this be manually tested?
- git pull branch `ft-add-structure-id-to-location-model-164266567` and check it out
- Follow this [setup](https://github.com/andela/mrm_api) to test.
- createLocation mutation as below:
![image](https://user-images.githubusercontent.com/12385482/53876066-114a0580-4017-11e9-907f-84cb04e9356b.png)
Since the create structure mutation hasn't been added yet, you need to manually add test office structure in `structure` table to get `structureId`.
#### What are the relevant pivotal tracker stories?
[#164266567](https://www.pivotaltracker.com/story/show/164266567)
